sodetlib/operations/translate_tes_parameters.py

```diff
@@ -0,0 +1,569 @@
+import numpy as np
+from scipy.interpolate import interp1d
+from scipy.optimize import curve_fit
+from sodetlib.operations.iv import IVAnalysis
+from sodetlib.operations.bias_steps import BiasStepAnalysis
+from tqdm.auto import trange
+
+
+import warnings
+warnings.filterwarnings('ignore')
+np.seterr(all="ignore")
+
+
+def translate_tes_params(iv_file, bs_file, save_iv=False, save_bs=False):
```

Comment:
Instead of passing file paths, can you accept pre-loaded iva and bsa dicts? This is more versatile and plugs in better with how we use it in the det_cal scripts.

Reply:
Returned values (iva2, bsa2) should start as deepcopies of the provided dicts.